tests/ui/too_long_first_doc_paragraph_module_level.fixed

```diff
@@ -0,0 +1,11 @@
+#![warn(clippy::too_long_first_doc_paragraph)]
+pub mod foo {
+
+    // in foo.rs
+    //! A very short summary.
+    //!
+//! A much longer explanation that goes into a lot more detail about
```

Comment:
The suggestion has some spacing inconsistency here

Reply:
My bad. I added spacing after the comment form variable.